src/Pages/Main/Head/HomeHead.js

```diff
@@ -0,0 +1,318 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+
+const BANNER_SLIDER = [
+  { id: 1, url: 'images/Main/slide1.webp' },
+  { id: 2, url: 'images/Main/slide2.jpeg' },
+  { id: 3, url: 'images/Main/slide3.jpeg' },
+  { id: 4, url: 'images/Main/slide4.webp' },
+  { id: 5, url: 'images/Main/slide5.webp' },
+];
+
+function HomeHead() {
+  const [isHover, setIsHover] = useState(false);
+
+  function handleToggle() {
+    setIsHover(isHover => !isHover);
+  }
+
+  const settings = {
+    dots: true,
+    infinite: true,
+    speed: 700,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    autoplay: true,
+    autoplaySpeed: 3000,
+    dotsClass: 'slickDots',
+    pauseOnHover: true,
+  };
+
+  return (
+    <Container>
+      <Row>
+        <BannerLeft
+          onMouseOver={() => handleToggle()}
+          onMouseOut={() => handleToggle()}
+        >
+          <StoryLink href="/">
+            <BannerImage transform={isHover} />
```

Comment:
콘솔창 확인하시면, Warning: Received `false` for a non-boolean attribute `transform`. 와 같은 에러를 확인하실 수 있습니다. 
해당 에러는 transform 이란 속성에 boolean 값을 줄 수가 없다는 의미를 가집니다.
이럴 경우 
```suggestion
            <BannerImage isHover={isHover} />
```
위와 같이 부여된 속성의 이름을 변경해 주시면 됩니다. 이에 따라 하위의 styled-component 도 변경해주세요!

Reply:
넵 수정 반영했습니다 감사합니다~